### DISCUSSION TITLE
Layout doesn't use PrimeVue components?

### DISCUSSION THREAD

**Discussion by crosswins** [primefaces/community#415] [PrimeVue Templates] [answered]
I purchased the Babylon template as an initial trial to integrate a layout into a Vue3 app I am building.
Having followed the guidance to bring in the relevant layout artefacts I see for example in the AppBreadcrumb.vue component that this page does not use the PrimeVue Breadcrumb component but rather is pure html markup (also using the global $route).

 I have a question: Is there a Template that implements the layout using the PrimeVue components?

My desire is to convert the layout pieces to TypeScript and composition API so that all views, including my own views, use the PrimeVue components. Is this reasonable or am I misunderstanding something?

**Comment by tugcekucukoglu** [comment, marked as answer]
Such a way was followed in order to avoid having to use PrimeVue component in layout components. If you want to translate them into PrimeVue components, we would like to help.
